package.json

```diff
@@ -36,8 +36,9 @@
     "env2": "^2.2.2",
     "express": "^4.17.1",
     "jsonwebtoken": "^8.5.1",
-    "pg": "^7.12.1",
     "morgan": "^1.9.1",
+    "pg": "^7.12.1",
+    "prop-types": "^15.7.2",
```

Comment:
Why you install the props-types here 😱
It's should be on the `client/package.json`

Reply:
delete it

Reply:
delete it